packages/site/package.json

```diff
@@ -10,10 +10,11 @@
     "clean": "rimraf dist",
     "prepublishOnly": "run-s clean build",
     "stage:build": "react-static build --staging",
-    "stage:deploy": "node scripts/deploy/surge.js",
     "prod:build": "react-static build",
-    "prod:deploy": "node scripts/deploy/aws.js",
-    "prod:serve": "serve -l 3000"
+    "prod:serve": "serve -l 3000",
+    "serve": "formideploy serve",
```

Comment:
Is it possible for us then to get rid of `serve` here?

Reply:
Just remove it?